### PR TITLE
check if docker is not installed and proceed

### PR DESCRIPTION
**Pull request by aquibbaig**
This is an intial PR that solves #71

### CODE REVIEW

File: lib/commands/deploy/docker.js [hunk 1]
```diff
@@ -3,29 +3,77 @@
...
+}
+
 exports.dockerize = () => {
   showBanner();
 
   setTimeout(() => {
     configFileExists();
 
+    if (!shell.which('docker')) {
+      installDocker();
```
Comment (jamesgeorge007) [inline lib/commands/deploy/docker.js]: Show a warning message such that docker is currently not installed.

File: lib/commands/deploy/docker.js [hunk 2]
```diff
@@ -3,29 +3,77 @@
...
+  })
+}
+
 exports.dockerize = () => {
   showBanner();
 
   setTimeout(() => {
     configFileExists();
 
+    if (!shell.which('docker')) {
```
Comment (jamesgeorge007) [inline lib/commands/deploy/docker.js]: Show a message such that `docker is not currently installed`

**Comment by aquibbaig** [comment]
@jamesgeorge007 I have made the changes you had asked. Can you check the logs and tell me why the build checks are failing?

**Comment by ajomadlabs** [comment]
@aquibbaig Have fixed the build issue

**Pull request merged**